frontend/lib/fhirRender.ts

```diff
@@ -15,6 +15,31 @@ export const patientName = (patient: Patient): string => {
     return [name.family, name.given?.join(" ")].filter(Boolean).join(", ");
 }
 
+export const findTelecom = (patient: Patient): Telecom => {
+    let phone = "Onbekend";
+    let email = "Onbekend";
+
+    if (patient.telecom) {
+        for (const telecom of patient.telecom) {
+            if (telecom.system === 'phone') {
+                phone = telecom.value || "Onbekend";
+            } else if (telecom.system === 'email') {
+                email = telecom.value || "Onbekend";
+            }
+        }
+    }
+    return new Telecom(phone, email);
+}
+
+
+export class Telecom {
```

Comment:
This is overkill. Especially since `Telecom` is an existing FHIR concept. The method above can simply return:

```
{ email, phone }
```